hypertrace-core-graphql-platform/build.gradle.kts

```diff
@@ -40,7 +40,7 @@ dependencies {
     }
 
     runtime("org.apache.logging.log4j:log4j-slf4j-impl:2.17.1")
-    runtime("io.grpc:grpc-netty:1.45.1")
```

Comment:
No netty change please - version needs to remain in sync with the rest of grpc deps